### DISCUSSION TITLE
postprocessing from this library and the native threejs postprocessing at the same time?

### DISCUSSION THREAD

**Discussion by ImaDoofus** [pmndrs/postprocessing#363] [Q&A]
The problem is I really like the god rays in this library but can't seem to make the SSAO look that good for my use. Is there a way to use postprocessing from threejs and postprocessing from this library as well?
Thanks :D

**Comment by vanruesc** [comment]
Hi,

this library isn't directly compatible with three's postprocessing code, but you can hack three's passes to account for API differences:

```js
import { SSAOPass } from "three";

// Haven't tested this, but it should work.
Object.assign(SSAOPass.prototype, {

	initialize() {},
	setRenderer() {},
	originalRender: SSAOPass.prototype.render,

	render(renderer, inputBuffer, outputBuffer, delta, maskActive) {

		this.originalRender(renderer, outputBuffer, inputBuffer, delta, maskActive);

	}

});
```
